mail_activity_permission/models/mail_activity_mixin.py

```diff
@@ -0,0 +1,58 @@
+# Copyright 2023 Hunki Enterprises BV
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from psycopg2.extensions import AsIs
+
+from odoo import api, fields, models
+from odoo.osv.expression import FALSE_DOMAIN
+
+
+class MailActivity(models.AbstractModel):
+    _inherit = "mail.activity.mixin"
+
+    activity_permission_ids = fields.One2many(
+        comodel_name="mail.activity",
+        compute="_compute_activity_permission_ids",
+        search="_search_activity_permission_ids",
+        groups="base.group_user",
+        context={"active_test": False},
+    )
+
+    @api.depends("activity_ids")
+    def _compute_activity_permission_ids(self):
+        MailActivity = self.env["mail.activity"].with_context(active_test=False)
```

Comment:
I think you should use another variable name, `MailActivity` is duplicated with the class name

Reply:
thanks, I've renamed the class instead to its canonical name

Reply:
Thanks, it is ok for me